frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.unit.spec.tsx

```diff
@@ -93,7 +93,11 @@ const setup = ({ card = TEST_CARD, result = TEST_RESULT }: SetupOpts = {}) => {
       <Route
         path="dashboard/:slug"
         component={() => (
-          <DashCardMenuConnected question={question} result={result} />
+          <DashCardMenuConnected
+            question={question}
+            result={result}
+            downloadsEnabled
```

Comment:
agree on this

Reply:
I was thinking of testing this directly at the PublicOrEmbedded{Question/Dashboard} level so that I can test the mix of `hide_download_buttons` and `downloads` hash parameters.
Would it be ok if I only tested at that level? this prop being passed down is kinda of an implementation detail

Reply:
Now it's hardcoded, I'm ok leaving it as is as long as you don't forget. In that case, it might be better to leave some comment like what you did here
https://github.com/metabase/metabase/blob/cad5e3f7f290aa83ddceb561063c31e97ad7ddbc/frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx#L411-L413